servicetalk-dns-discovery-netty/src/main/java/io/servicetalk/dns/discovery/netty/DefaultDnsServiceDiscovererBuilder.java

```diff
@@ -233,16 +221,11 @@ public DefaultDnsServiceDiscovererBuilder ioExecutor(final IoExecutor ioExecutor
      * @return a new instance of {@link DnsClient}.
      */
     DnsClient build() {
-        DnsClient rawClient = new DefaultDnsClient(
+        final DnsClient rawClient = new DefaultDnsClient(
                 ioExecutor == null ? globalExecutionContext().ioExecutor() : ioExecutor, minTTLSeconds, ndots,
                 invalidateHostsOnDnsFailure, optResourceEnabled, queryTimeout, dnsResolverAddressTypes,
                 dnsServerAddressStreamProvider);
-        DnsClientFilterFactory rawFilterFactory = filterFactory;
-        if (applyRetryFilter) {
```

Comment:
As we do not use `RetryingDNSClientFilter` anymore, we can delete the class and remove references from the tests